opencsp/app/sofast/test/test_SpatialOrientation.py

```diff
@@ -68,6 +71,28 @@ def test_rotation_ring_2(self):
         I_calc = self.so.r_cam_screen * self.so.r_optic_cam * self.so.r_screen_optic
         np.testing.assert_allclose(I_exp, I_calc.as_matrix(), atol=1e-9, rtol=0)
 
+    def test_io_oriented_optic(self):
+        file = join(self.save_dir, 'test_spatial_orientation_oriented_optic.h5')
+        # Save
+        self.so.save_to_hdf(file)
+        # Load
+        ori = SpatialOrientation.load_from_hdf(file)
+        # Check optic is oriented
+        np.testing.assert_equal(ori.optic_oriented, True)

```

Comment:
Exactly! Like this! :)

Note that if you're comparing two standard python objects, you can also use `self.assertEqual(a, b)`, which in my opinion produces easier-to-read results in the case that the two instance's aren't equal.  In reality, it's just a matter of opinion which one you like more.

Example of `np.testing.assert_equal` and `unittest.TestCase.assertEqual`
![image](https://github.com/sandialabs/OpenCSP/assets/144833583/f71ef602-6cbb-485e-b1e4-d28d582b8243)

Reply:
I like that method a lot better. Updated!